app/backend/submissions/tests.py

```diff
@@ -322,7 +379,7 @@ def test_no_city_on_legacy(self):
             'well': well.well_tag_number
         }
 
-        # NOTE: this test previously asserted that a 
+        # NOTE: this test previously asserted that a
```

Comment:
```suggestion
```

I think I accidently left this half-finished comment (I moved it to the docstring but left a few words down here)